include/spla-cpp/SplaLibrary.hpp

```diff
@@ -30,6 +30,8 @@
 
 #include <spla-cpp/SplaConfig.hpp>
 #include <spla-cpp/SplaRefCnt.hpp>
+#include <boost/compute/device.hpp>
+#include <boost/compute/system.hpp>
 #include <memory>
```

Comment:
@Glebanister  `Boost` related stuff is not allowed in `include/spla*` interface headers.
Library interface must be clear and include only `std` dependencies.
All boost stuff must be done in `sources/` folder.